### PR TITLE
Add newTabWarning to Footer and update docs

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This PR adds the ability to add a warning at the top of the Footer to warn users that links will open in a new tab and updates docs to reflect this.
It also:
- updates the Footer docs to add a recent new param (crest)
- fixes issues setting `footerWarning` and `copyrightDeclaration` using pageConfig
- removes unused param `serviceLinks` from pageConfig
- updates pageConfig docs for Footer
- renames `poweredByONS` to `poweredBy` in docs

### How to review 
- Check footer with warning line looks as it should
- Check docs are up to date

### CODE REVIEW

**Comment by rmccar** [comment]
[Add 'opens in new tab' lead-in line to footer](https://trello.com/c/pD7BrNQ4/748-add-opens-in-new-tab-lead-in-line-to-footer)

**Comment by jrbarnes9** [comment]
Inline footer links seem to be stacking too soon.  Should stack to a single column until they need to.  This example shows there's plenty of room to allow them to sit inline at this width.
Screenshot 2020-10-22 at 17 05 35

File: src/components/footer/_macro.njk [hunk 1]
```diff
@@ -94,6 +94,10 @@
                         </div>
                     {% endif %}
 
+                    {% if params.newTabWarning is defined and params.newTabWarning %}
+                        <p class="u-fs-r footer__new-tab-warning">{{ params.newTabWarning }}</p>
```
Comment (jrbarnes9) [inline src/components/footer/_macro.njk:98]: From the design, it looks like the lead in line should be `u-fs-s`
image

File: src/components/footer/_macro.njk [hunk 2]
```diff
@@ -94,6 +94,10 @@
                         </div>
                     {% endif %}
 
+                    {% if params.newTabWarning is defined and params.newTabWarning %}
+                        <p class="u-fs-s footer__new-tab-warning">{{ params.newTabWarning }}</p>
```
Comment (jrbarnes9) [inline src/components/footer/_macro.njk:98]: If you wrap this in `grid__col` and add `u-mb-m` to the `<p>`, you don't need any css.

Comment (jrbarnes9) [inline src/components/footer/_macro.njk:98]:
```suggestion
                        <p class="footer__new-tab-warning u-fs-s u-mb-m">{{ params.newTabWarning }}</p>
```

**Review by jrbarnes9 — APPROVED** [review]

**Pull request merged**